build.gradle.kts

```diff
@@ -33,6 +33,28 @@ allprojects {
         mavenLocal()
         google()
         mavenCentral()
+        jcenter {
```

Comment:
can we just leave fresco, because it would require some extra work on Buck.

Reply:
can you elaborate a little bit? This PR was green before public and internal, but I needed to rebase and in the mean time https://github.com/facebook/react-native/commit/70da64094608f5f2e3c554ed719e9aad624e3459 happened. Is that something that should partially be reversed? Afaik everything seems to be green atm, except for test_docker job. Does that relate to the extra Buck work?

Reply:
I would recommend to not touch other dependencies unless necessary. Here is Fresco in Buck, you need to update in here too https://github.com/facebook/react-native/blob/dc80b2dcb52fadec6a573a9dd1824393f8c29fdc/ReactAndroid/src/main/libraries/fresco/fresco-react-native/BUCK

Reply:
Also it would be nice to remove below lines in https://github.com/facebook/react-native/blob/master/template/android/app/build.gradle and https://github.com/facebook/react-native/blob/master/packages/rn-tester/android/app/build.gradle because it's unnecessary with the Flipper bump. 

```Gradle
debugImplementation("com.facebook.fresco:stetho:2.3.0")
exclude group:'com.facebook.fresco', module:'stetho'
```

Reply:
test_docker failure is not related to your change. I can open a PR very soon

Reply:
where did you get the hashes from in the previous update? Sorry I don't really grok the RN stack :). I merely wanted to bump the Flipper version but the rabbit hole goes a bit deeper than anticipated 😢